CONTRIBUTING.md

```diff
@@ -78,9 +78,13 @@ defined at <https://developer.github.com/v3/repos/hooks/> live in
 
 (These notes are mostly only for people merging in pull requests.)
 
-**First and foremost, try to maintain a clean, linear git history.**  With very
-few exceptions, running `git log` should not show a bunch of branching and
-merging.
+**Verify CLAs.** CLAs must be on file for the pull request submitter and commit
+author(s). Google's CLA verification system should handle this automatically
+and will set commit statuses as appopriate. If there's ever any question about
+a pull request, ask [willnorris](https://github.com/willnorris).
+
+**Always try to maintain a clean, linear git history.** With very few
+exceptions, running `git log` should not show a bunch of branching and merging.
 
 Never use the GitHub "merge" button, since it always creates a merge commit.
```

Comment:
Maybe this statement should be updated/revisited now that you can merge using "rebase and squash" strategy, which means the "since it always creates a merge commit" statement is no longer correct.